### PR TITLE
Update-xod-showcase

### PR DESCRIPTION
**Pull request by gabbapeople**
Add the article series about programming mBot.

### CODE REVIEW

File: docs/showcase.yaml [hunk 1]
```diff
@@ -23,8 +23,21 @@
...
+- title: XOD powered mBot
+  image: ./__img__/mbot-pid.jpg
+  url: "https://medium.com/xodlang/xod-and-pid-powered-self-balancing-mbot-6c420600271e"
+  author: Max Danilin
+  description: |
+    Article series (
+    <a
+      href="https://medium.com/xodlang/how-to-program-mbot-with-xod-pid-controller-c3e310f8eceb">Hand follower mBot,</a><a
+      href="https://medium.com/xodlang/xod-powered-line-follower-mbot-2ae4a4862a9e">Line follower mBot,</a><a
+      href="https://medium.com/xodlang/xod-and-pid-powered-self-balancing-mbot-6c420600271e">Self-balancing mBot,</a><a
```
Comment (nkrkv) [inline docs/showcase.yaml]: Oops. You’ve forgotten an orphan `<a` tag on the end. And the final comma should not be there. This line should be:

```
      href="https://medium.com/xodlang/xod-and-pid-powered-self-balancing-mbot-6c420600271e">Self-balancing mBot</a>
```

File: docs/showcase.yaml [hunk 2]
```diff
@@ -23,8 +23,21 @@
       href="https://medium.com/@tdubya17/hacking-nodes-with-c-part-four-3d84723d671d">4</a>
     )
 
+- title: XOD powered mBot
+  image: ./__img__/mbot-pid.jpg
+  url: "https://medium.com/xodlang/xod-and-pid-powered-self-balancing-mbot-6c420600271e"
```
Comment (nkrkv) [inline docs/showcase.yaml]: I think linking to the first part of the series is preferable.

**Comment by gabbapeople** [comment]
@nkrkv @knopki Check it pls.

File: docs/showcase.yaml [hunk 3]
```diff
@@ -23,8 +23,21 @@
...
+  author: Max Danilin
+  description: |
+    Article series (
+    <a
+      href="https://medium.com/xodlang/how-to-program-mbot-with-xod-pid-controller-c3e310f8eceb">Hand follower mBot,</a><a
```
[the remark this answers is not in the document]
Comment (nkrkv) [reply docs/showcase.yaml]: Yeah. It’s a flaw in our engine which places some ugly unexpected whitespace before a comma if it is placed outside of the anchor.

**Pull request merged**